rmc-docs/src/getting-started.md

```diff
@@ -1,9 +1,17 @@
 # Getting started with RMC
 
-Hello, World!
+RMC is a Rust verification tool based on _model checking_.
+With RMC, you can ensure that broad classes of problems are absent from your Rust code by writing _proof harnesses_, which are broadly similar to tests (especially property tests.)
+RMC is especially useful for verifying unsafe code in Rust, where many of the language's usual guarantees can no longer be checked by the compiler.
```

Comment:
Maybe add that RMC can also find panics in safe Rust code (such as slice indexing errors) and user-added assertions.